dashboard/test/ui/features/step_definitions/check_finish_button.rb

```diff
@@ -0,0 +1,41 @@
+# Helper steps for dance party levels
+free_play_level_urls = {
+  'blockly' => {
+    'Dance Party' => 'http://studio.code.org/s/dance/stage/1/puzzle/13?noautoplay=true',
```

Comment:
If possible, it's good to use levels from `/s/allthethings`.  Not crucial to change here, but we try not to change those levels because they get used for tests like these.

Reply:
I looked there first, but as far as I could tell none of those levels mirror the free play level - none of them render a 'finish' button after the 'play' button is pressed. The JIRA ticket for this issue was created specifically to address a complaint about the finish button, so I tracked down the places in our application that do that.

Maybe it'd be possible to add levels that mirror free play levels in this way to '/s/allthethings'?

Reply:
Oh interesting.  Good investigation, and no worries.  This works fine.